naming-conventions.md

```diff
@@ -0,0 +1,79 @@
+# Naming Conventions
```

Comment:
at the end of the project you don't want this file in there

Reply:
Yeah, I've taken it out now